mock_sink.go

```diff
@@ -6,6 +6,7 @@ type mockSink struct {
 	Gauges   map[string]uint64
 }
 
+// Returns a Mock Sink that flushes stats to in memory maps.
```

Comment:
golint will complain here. All comments should begin with the symbol name or an article (`the`, `a`, `an`). So this could be rephrased as `// NewMockSink returns a Sink that flushes to an in-memory map`